core/templates/dev/head/components/summary_tile/ExplorationSummaryTileDirective.js

```diff
@@ -130,7 +131,17 @@ oppia.directive('explorationSummaryTile', [
             } else {
               var result = '/explore/' + $scope.getExplorationId();
               if ($scope.getCollectionId()) {
-                result += ('?collection_id=' + $scope.getCollectionId());
+                result = UrlService.addParams(result, 'collection_id',
+                  $scope.getCollectionId())
+              }
+              if ($scope.getParentExplorationIds()) {
+                var parentExplorationIds = $scope.getParentExplorationIds();
+                var i = 0;
```

Comment:
Drop this line.

Reply:
Done, I added that, when I was trying a while loop and forgot to remove it.